### ISSUE TITLE
Translations of the Mumble website

### ISSUE THREAD

**Issue by mkrautz** [mumble-voip/mumble-www#4]
At some point, we should probably look into translating the website.

How do we do that with Hugo?

How will our URLs look?

Etc.

**Comment by Natenom** [comment]
https://gohugo.io/content-management/multilingual/

The URL can be set in the metadata section of every page in hugo.

For example:
* EN: www.mumble.info/topic-1
* DE: www.mumble.info/de/thema-1
...

**Comment by hacst** [comment]
Imho we haven't had good experiences with the translations in the wiki. It lead to very inconsistent content depth and accuracy. Are we sure we want the burden of maintaining translations? Imho half-assing those is worse than not having them to begin with.

**Comment by Kissaki** [comment]
That is definitely true for the wiki, for the few guides/pages we have translations for.

It may be more viable for the website, with [more] static content. Should be avoided for Blog though, probably. Maybe release notes would be viable.

Looking at the pages, there is only the Home and Downloads page that should/could be translated. For Contribute users have to speak English to contribute anyway, and blog is too much effort.

**Comment by preterive** [comment]
I thing a german Homepage would be great

**Comment by toby63** [comment]
In case someone wants to make a pull request for translations, in which folder should they be placed?
Maybe you could open a branch for that :thinking:.

**Comment by Kissaki** [comment]
I do not think we are ready for this yet, and there are more pressing matters right now to tackle.

**Comment by toby63** [comment]
Ok.
